responses.py

```diff
@@ -1,26 +1,17 @@
 import random
 
 
-
 def get_response(message: str) -> str:
-    p_message = message.lower()
-    a = ['Rock', 'Paper', 'Scissors']
-    if p_message == 'hello':
-        return 'Hey there!'
-
-    if message == 'roll':
-        return str(random.randint(1, 6))
- 
-    if message == 'Rock':
-        return str(random.choice(a))
-
-    if message == 'Paper':
-        return str(random.choice(a))
-
-    if message == 'Scissors':
-        return str(random.choice(a))
-
 
+    p_message = message.lower()
 
+    rock_paper_scissors = ["rock", "paper", "scissors"]
 
+    if p_message == "hello":
+        return "Hey there!"
 
+    if p_message == "roll":
+        return str(random.randint(1, 6))
+ 
+    if p_message in rock_paper_scissors:

```

Comment:
If we make all our values in the `rock_paper_scissors` list lowercase strings, then we can simplify the 3 if statements into one.